src/Wpf.Ui/Controls/ListView/GridViewColumn.cs

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+// Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+// All Rights Reserved.
+
+using System.Reflection;
+
+namespace Wpf.Ui.Controls;
+
+public class GridViewColumn : System.Windows.Controls.GridViewColumn
```

Comment:
Is the **Controls/ListView** directory the right place for these classes? Maybe **Controls/GridView** would be more appropriate